catalog/plugin/pom.xml

```diff
@@ -33,5 +33,6 @@
         <module>catalog-plugin-expirationdate</module>
         <module>catalog-plugin-content-uri</module>
         <module>catalog-plugin-clientinfo</module>
+        <module>catalog-plugin-metacardingest-network</module>
```

Comment:
Too specific. That's only one of the criteria we offer to write conditions against.